packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -17,13 +17,19 @@ import {
 
 const Section = classed('div', 'flex flex-col');
 
-export function AuthenticationBanner(): ReactElement {
+type AuthenticationBannerProps = {
+  children?: ReactNode | ReactNode[];
+};
+export function AuthenticationBanner({
+  children,
+}: AuthenticationBannerProps): ReactElement {
```

Comment:
I think we use just this so we won't have to introduce one. Just import this through React.

```suggestion
}: PropsWithChildren): ReactElement {
```

Reply:
Nice, didn't know this was a type!